<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>date_mode_widget</class>
 <widget class="QWidget" name="date_mode_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
background-color:transparent;
color:rgb(122, 122, 122);}
QLabel:hover{
background-color:rgb(38, 38, 38);
color:rgb(255, 255, 255);}

QPushButton{
background-color:transparent;
color:rgb(122, 122, 122);
border:none;}
QPushButton:hover{
background-color:transparent;
border:1px solid rgb(255, 170, 0);
border-left:none;
border-right:none;
border-top:none
}
QWidget{
background-color:rgb(32, 32, 32);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="date_header_frame">
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:transparent;
border:none;
}
QPushButton:hover{
background-color:transparent;
border:1px solid darkgrey;
}
QLabel{
color:skyblue;}
QLabel:hover{
color:rgb(0, 85, 127);
border:2px solid rgb(0, 58, 0);
	border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QPushButton" name="DateManasIconpushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../model/icon/icon.qrc">
          <normaloff>:/icons/MANAS_ICON.png</normaloff>:/icons/MANAS_ICON.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="date_title_label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:skyblue;&quot;&gt;Date Calculation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="date_category_frame">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="3" column="0">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="from_date_calendar_btn">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>show calendar</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../model/icon/icon.qrc">
          <normaloff>:/icons/calendar.svg</normaloff>:/icons/calendar.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="date_from_label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_11">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_12">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="4">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QComboBox" name="date_category_cb">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;
color:rgb(255, 255, 255);</string>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QDateEdit" name="initial_dateEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:skyblue;
border:none;</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="date_main_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QStackedWidget" name="date_main_stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="date_difference_widget">
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="date_to_label">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>To</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" alignment="Qt::AlignLeft">
              <widget class="QPushButton" name="date_to_calendar_btn">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../model/icon/icon.qrc">
                 <normaloff>:/icons/calendar.svg</normaloff>:/icons/calendar.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <spacer name="verticalSpacer_9">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>23</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="8" column="0">
              <spacer name="verticalSpacer_18">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>30</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="0">
              <spacer name="verticalSpacer_8">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0" alignment="Qt::AlignLeft">
              <widget class="QLabel" name="date_difference_title_label">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Difference</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="2">
              <widget class="QLabel" name="date_difference_label">
               <property name="font">
                <font>
                 <family>Calibri Light</family>
                 <pointsize>18</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Difference Date</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QDateEdit" name="date_to_label_date">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color:skyblue;
border:none;</string>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
               <property name="currentSectionIndex">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="all_days">
               <property name="styleSheet">
                <string notr="true">color:rgb(255, 255, 255)</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="date_add_soubstract_widget">
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QFrame" name="date_sum_diff_frame">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
QRadioButton{
background-color:transparent;
color:rgb(122, 122, 122);
border:none;}
QRadioButton:hover{
background-color:transparent;
color:rgb(255, 255, 255);
border:none;}
QComboBox{
background-color:rgb(32, 32, 32);
color:rgb(255, 255, 255);
border:1px solid rgb(159, 159, 159);
}
QComboBox:hover{
background-color:rgb(32, 32, 32);
color:rgb(255, 255, 255);
border:1px solid skyblue;
border-radius:2px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>2</number>
             </property>
             <item row="0" column="2">
              <widget class="QRadioButton" name="dare_substract_radioButton">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>substract</string>
               </property>
               <property name="text">
                <string>Substract</string>
               </property>
               <property name="icon">
                <iconset resource="../../model/icon/icon.qrc">
                 <normaloff>:/icons/minus.svg</normaloff>:/icons/minus.svg</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="date_years_label">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Years</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="date_month_label">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Months</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="date_add_radioButton">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>add</string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
               <property name="icon">
                <iconset resource="../../model/icon/icon.qrc">
                 <normaloff>:/icons/plus.svg</normaloff>:/icons/plus.svg</iconset>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QComboBox" name="date_years_combo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="toolTip">
                <string>input years</string>
               </property>
               <property name="currentText">
                <string/>
               </property>
               <property name="maxVisibleItems">
                <number>15</number>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
               <property name="minimumContentsLength">
                <number>0</number>
               </property>
               <property name="frame">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QComboBox" name="date_month_combo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="toolTip">
                <string>input month</string>
               </property>
              </widget>
             </item>
             <item row="4" column="4">
              <widget class="QComboBox" name="date_days_combo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="toolTip">
                <string>input days</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="date_result_title_label">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Date</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QLabel" name="date_days_label">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Days</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0" colspan="5">
              <widget class="QLabel" name="date_result_label">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>18</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color:rgb(243, 243, 243)</string>
               </property>
               <property name="text">
                <string>Final Date</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <spacer name="verticalSpacer_14">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer_13">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="7" column="2" colspan="3">
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="6" column="0">
              <spacer name="verticalSpacer_17">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="8" column="0">
              <spacer name="verticalSpacer_16">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>42</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="5" column="0">
              <spacer name="verticalSpacer_15">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="1">
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="3">
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../model/icon/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>from_date_calendar_btn</sender>
   <signal>clicked()</signal>
   <receiver>initial_dateEdit</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
